client/src/components/History/HistoryViews/JobHistoryModal.js

```diff
@@ -98,6 +130,9 @@ function JobHistoryContent({ jobDetails, jobDisplay, jobName }) {
   if (jobDetails.status !== 'FINISHED')
     configDataObj['Temporary Working Directory'] = jobDetails.workPath;
 
+  const jobFinished =
+    jobDetails.status === 'FINISHED' || jobDetails.status === 'FAILED';
```

Comment:
We can use the `isTerminalState` function in `jobsUtil.js`, which also includes `STOPPED`